### DISCUSSION TITLE
Pulse parametrization

### DISCUSSION THREAD

**Discussion by Tanguy-Chamaillard** [qucontrol/krotov#108] [Q&A] [answered]
Hi! 

I've started to work with the Krotov package and wanted to know if/when you are planning to release the parametrization feature to limit the amplitude of the controls? I tried to modify the controls in the same way as applying spectral constraints but it doesn't work well. So I would be very grateful if you know a way to apply amplitude constraints in the meantime.

Thanks a lot in advance,
Tanguy

**Comment by goerz** [comment, marked as answer]
You can try the code in https://github.com/qucontrol/krotov/pull/92

This will be merged at some point, but I can't give you a timeline for when

**Reply by Tanguy-Chamaillard** [reply]
Great, thank you for your answer!
